ui/jstests/test-learning-resource.jsx

```diff
@@ -554,8 +561,56 @@ define(['QUnit', 'jquery', 'react', 'lodash', 'learning_resources',
     function(assert) {
       var div = document.createElement("div");
       assert.ok($(div).html().length === 0);
-      LearningResources.loader("repo", "1", function() {}, function() {}, div);
+      LearningResources.loader(
+        "repo", "1", function() {}, function() {}, function() {}, false,
+        false, div
+      );
       assert.ok($(div).html().length > 0);
     }
   );
+
+  QUnit.test('Assert learning resource panel does close',
+    function (assert) {
+      var done = assert.async();
+      var closeLearningResourcePanelCount = 0;
+      var closeLearningResourcePanel = function() {
+        closeLearningResourcePanelCount += 1;
+      };
+
+      var Container = React.createClass({
+        getInitialState: function() {
+          return {closeLRPanel: false};
+        },
+        render: function() {
+          return <LearningResourcePanel
+            repoSlug="repo"z
+            ref={this.afterMount}
+            closeLearningResourcePanel={this.props.closeLearningResourcePanel}
+            closeLRPanel={this.state.closeLRPanel}
+            learningResourceId="1" />;
+        },
+        afterMount: function () {
+          waitForAjax(9, function () {
+            assert.ok(
+              closeLearningResourcePanelCount > 0, "Close panel was called"
+            );
+            done();
+          });
+        }
+      });
+
+      var afterMount = function (component) {
+        component.setState({closeLRPanel: true}, function () {
+          component.forceUpdate(function() {
+            assert.equal(component.state.closeLRPanel, true);
```

Comment:
line 597 will be called at the end, because each time we change stage if container new LR panel renders.